### DISCUSSION TITLE
Implicit solvation geometry optimization fails for higly charged POM

### DISCUSSION THREAD

**Discussion by codecollider** [grimme-lab/xtb#769] [Q&A]
Hi,

I´m trying to geometry optimize a singly protonated Keggin-Dawson POM structure (P2W18O62H) in the charge state -5 (coords are shown below). The geometry optimization is running fine when using `xtb coord --gfn 1 --chrg -5 --opt`, but it is not working when using the implicit solvation model, e.g running.

`xtb coord  --gfn 1 --alpb water --opt --chrg -5 `

I tried alpb and gbsa with water or methanol and both geometry optimizations do not converge. The first optimization steps are working, but at some point the energy goes crazy. I also tried toluene and in this case it is working fine, maybe because it is less polar (?). 

Setting etemp higher also didn´t solve the problem. I´m using GFN 1, because GFN 2 does not work for the POMs (the cage collapses). I there anything I can try beside of what I did?

Thanks for the help and have nice weekend, 
Marco

83
 energy: -345.346588919820 gnorm: 0.000586692727 xtb: 6.5.1 (579679a)
W           -0.05414087995027        6.13859895508601       32.24266257970957
W           -0.12265682740682        5.86153053600755       35.51603289244861
W           -0.03592852356779        3.17304060652535       33.63937925375328
W            2.93568623968474        2.32137395340652       31.81117629555352
W            2.92428027365558        5.34661761132816       30.37966470381880
W            2.81812138933875        8.30486303850834       32.45016279115233
W            2.74866427299812        8.01716214281666       35.79932114451916
W            2.78229816988649        4.74434051967092       37.33199139285805
W            2.88113590354320        2.00528786214443       35.40943228174974
W            6.52360244561505        2.08864825626183       35.53007516203842
W            6.54501379000732        2.61861508116948       31.98171161350168
W            6.57560099483684        5.46413087450050       30.40761990605730
W            6.46261862685123        8.39225699357474       32.51899865520378
W            6.39954902474657        8.12909016520776       35.92796142201621
W            6.42935551538831        4.83232954854627       37.44129172270882
W            9.36457532027707        6.06413486686656       35.74167344521724
W            9.46999497178439        3.37535041285029       33.86220346612902
W            9.42658065773987        6.33541005841018       32.44312035830787
P            2.85565338124891        5.10412221868781       33.85723633857441
P            6.50084663430476        5.14395470717987       33.93452330099550
O           -1.22296613994080        6.83545255189365       31.18571617128805
O            1.20487355141729        5.47462603555795       31.06191605631667
O           -0.72678892297167        4.41408056127957       32.44295186335027
O           -0.79396888919739        6.50760382619815       33.90685139373433
O            1.13140822814111        7.52803795234835       32.49858288258925
O            1.27076272466977        5.06653762374411       33.81468881763205
O           -1.33711430886886        6.37251509094399       36.62659311827197
O            1.06696103251040        7.27030257617617       35.54487818158567
O           -0.77822547528528        4.19833424865944       34.99405073284991
O            1.08886309059454        5.00164231346972       36.60891717577421
O           -1.18599369822988        1.89654418957246       33.50986563361904
O            1.21818736336920        2.69878952218707       32.35409077221575
O            1.17951711697172        2.49574869711949       34.85168973626003
O            2.67215563101681        0.79031268591524       31.06138206508965
O            4.79884260956113        2.37007796699957       31.58691816476467
O            2.76972940432058        3.48832061384847       30.37777982209272
O            3.23459074334229        1.88749768392493       33.58367067007615
O            3.35620884683202        4.48885763926620       32.54442402073339
O            2.58816425442269        5.73690250846863       28.73506485137034
O            3.17367523200722        6.98992726989430       31.18516717292866
O            4.77211017081738        5.19594866183988       30.24191129076184
O            2.43200136406260        9.69458521076735       31.50688422007194
O            4.64061642442527        8.57529148509737       32.59098130870948
O            2.58509938675820        8.92676971510656       34.18618847192820
O            3.27002071435451        6.56843667455110       33.99150801003643
O            2.32101166029873        9.22681224159158       36.94989215126662
O            4.58094279326772        8.30457330688189       35.78357345691959
O            3.05597955165839        6.50580335157237       36.82816709087960
O            2.37086198393560        4.85159083270440       39.00203047347109
O            2.62931386911554        2.91818190753809       37.00685284727793
O            4.61647911787486        4.56807018488398       37.51793824557321
O            3.29508862418391        4.27624889514562       35.06701675839123
O            2.53847954318640        0.37722115812998       35.86007419827074
O            4.70594214796688        1.97886682730503       35.71942274867686
O            6.92776762311419        0.47257984344113       35.96885262765304
O            8.23081480316275        2.64602438133666       35.01157678698205
O            6.68611091452604        3.00857372104935       37.12080158072131
O            6.25337935375517        2.01649121117523       33.67144257056394
O            6.03719419833980        4.33063249625577       35.13549541730585
O            6.93216034384239        0.95957674509294       31.21007243112032
O            6.81425037551850        3.59929937106011       30.45196531618834
O            8.26901293090502        2.86562347477390       32.50781802243880
O            6.10961513646858        4.43476131851790       32.60665521176269
O            6.98342668564613        5.86236866339056       28.78182500757080
O            8.24513234951111        5.62049559223036       31.20911099299499
O            6.24025855371635        7.08750061388888       31.20224494632710
O            6.84496662656551        9.81544059007181       31.62714448998566
O            8.17968561598137        7.67158302044525       32.64123153635623
O            6.60375735754105        8.97111904246550       34.28477325207822
O            5.99089597262473        6.56708451572115       33.94040080481954
O            6.73807793147476        9.36587889321322       37.07965173968178
O            8.11533690067663        7.41789891978578       35.71772568385139
O            6.12284849919928        6.58693360222220       36.92289721065018
O            6.77047110489783        4.95692360940694       39.12517719672964
O            8.14512883613961        5.14763746339372       36.77247441501781
O           10.51428370993881        6.62532178969173       36.89539418183929
O           10.11188891505777        4.42587051044103       35.24037577436268
O           10.06004174478861        6.72321304665929       34.14759229657180
O            8.07634496483057        5.17413450085425       33.95148239935391
O           10.66392308967842        2.13838916353765       33.76868492693821
O           10.15499734450090        4.63388932926518       32.68233848479431
O           10.62006497031260        7.07396178857442       31.44454712906283
H            6.25848201971418        0.37542236270615       30.86836826518647

**Comment by haneug** [comment]
I would say it is impressive that GFN1-xTB converges at all for this system, as metal oxides are a typical weakness for GFNn-xTB methods as they were not parameterized for metal oxides. Maybe [H0 tuning](https://xtb-docs.readthedocs.io/en/latest/sp.html?highlight=H0#h0-tuning) for W and O could help with convergence? You can generate a DFT gradient as a reference to optimize the pair parameter. See this [publication](https://onlinelibrary.wiley.com/doi/10.1002/anie.202102679) for a nice example of where H0 tuning can help. Alternatively, you can use GFN0-xTB (no SCC so no convergence problems).

**Reply by codecollider** [reply]
Thanks, I´m looking into this!

**Comment by MtoLStoN** [comment]
You could try our dd-COSMO implementation with xtb version 6.6.0. For example, the provided POM structure seems to converge to a reasonable geometry using ``xtb pom.xyz --chrg -5 --gfn 1 --cosmo water --opt``.
